packages/object-deep-map-values/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = deepMapValues;
+
+function deepMapValues(obj, fn) {
+  return Object.keys(obj).reduce(function(acc, key) {
```

Comment:
Can you replace `reduce` with a for loop? Current guidelines are to treat all utilities as potentially hot code and so micro-optimize accordingly. (Functions that include deep object traversal could be particularly susceptible to the effect of multiple function calls, specially in non-optimized browsers)

Reply:
Done